src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -2011,5 +2011,99 @@ private static bool TryFindFirstMatchedLane(Vector128<byte> mask, Vector128<byte
             matchedLane = BitOperations.TrailingZeroCount(selectedLanes) >> 2;
             return true;
         }
+
+        public static void Reverse(ref byte buf, nuint length)
+        {
+            int numBytes = (int)length;
+            ref byte first = ref buf;
+            ref byte last = ref Unsafe.Add(ref Unsafe.Add(ref first, numBytes), -1);
```

Comment:
Semantically, using [Subtract](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.unsafe.subtract?view=net-6.0) would be better.
```suggestion
            ref byte last = ref Unsafe.Subtract(ref Unsafe.Add(ref first, numBytes), 1);
```